bin/uwsgi.ini

```diff
@@ -0,0 +1,8 @@
+[uwsgi]
+module = basket.wsgi:application
+http = :8000
+master = true
+workers = 8
```

Comment:
If it helps, over in Nucleus I've left this out and set `--workers` in run-prod.sh so we can tune it with env vars. Might it be worth doing that here, in case we've got WSGI_NUM_WORKERS set to something special in k8s?

```uwsgi --ini /app/bin/uwsgi.ini --workers "${WSGI_NUM_WORKERS:-4}"```

Reply:
I saw that. I will do the same 👍